relay/pkg/btc/btc.go

```diff
@@ -16,6 +16,9 @@ type Handle interface {
 
 	// GetHeaderByDigest returns the block header for given digest (hash).
 	GetHeaderByDigest(digest Digest) (*Header, error)
+
+	// GetBlockCount returns the number of blocks in the longest blockchain
+	GetBlockCount() (int64, error)
```

Comment:
`btc/local` chain doesn't implement this method and this cause failure of the CI job